elb_log_ingestor/elb_log_fetcher.py

```diff
@@ -191,7 +193,10 @@ def enqueue_log(self, count: int = 1) -> str:
         # push them to the queue && move them
         filenames = self.input_dir.glob("*.log")
         for filename in list(filenames)[:self.file_batch_size]:
-            processing_name = self.mark_log_processing(filename)
+            try:
```

Comment:
This protects a case where we get multiple instances grabbing the same file at the same time.